src/Symfony/Component/Lock/Store/RedisStore.php

```diff
@@ -244,12 +244,7 @@ public function exists(Key $key)
         return (bool) $this->evaluate($script, (string) $key, [microtime(true), $this->getUniqueToken($key)]);
     }
 
-    /**
-     * Evaluates a script in the corresponding redis client.
-     *
-     * @return mixed
-     */
-    private function evaluate(string $script, string $resource, array $args)
+    private function evaluate(string $script, string $resource, array $args): mixed
```

Comment:
function is missing a return statement at the end as psalm says

Reply:
also there is a double return in line 272/274

Reply:
good catch, fixed in https://github.com/symfony/symfony/pull/42065